commit 278876d6fb9df8ae1d48063647af508b87d06213
Author: nobrakal <nobrakal@gmail.com>
Date:   Thu Jan 16 08:41:29 2014 +0100

    Update opencs.desktop
    
    Fix bad output of desktop-file-validate for opencs.dektop

diff --git a/files/opencs.desktop b/files/opencs.desktop
index 80afa26..638de6e 100644
--- a/files/opencs.desktop
+++ b/files/opencs.desktop
@@ -3,7 +3,7 @@ Type=Application
 Name=OpenMW Content Editor
 GenericName=Content Editor
 Comment=A replacement for the Morrowind Construction Set.
-Keywords=Morrowind;Construction Set;Creation Kit Editor;Set;Kit
+Keywords=Morrowind;Construction Set;Creation Kit Editor;Set;Kit;
 TryExec=opencs
 Exec=opencs
 Icon=opencs
